cmd/mesh/manifest-versions.go

```diff
@@ -105,8 +104,26 @@ func getVersionCompatibleMap(versionsURI string, binVersion *goversion.Version,
 		}
 	}
 	if myVersionMap == nil {
-		l.logAndFatal("This operator version ", binVersion.String(),
-			" was not found in the global manifestVersions map.")
+		return nil, fmt.Errorf("this operator version %v was not found in the global manifestVersions map", binVersion.String())
```

Comment:
nit: use %s for string. %v is ok but not idiomatic.

Reply:
Fixed.